bucket/tf2-hud-editor.json

```
@@ -0,0 +1,42 @@
+{
+    "version": "2.5",
+    "description": "Install and customize your favorite custom Team Fortress 2 HUDs",
+
+    "homepage": "https://www.editor.criticalflaw.ca",
+    "license": {
+        "identifier": "MIT",
+        "url": "https://spdx.org/licenses/MIT.html"
+    },

```

Standard license identifiers don't need a URL:

```suggestion
    "license": "MIT",
```